dissect/target/plugins/os/unix/bsd/citrix/_os.py

```diff
@@ -135,22 +139,28 @@ def users(self) -> Iterator[UnixUserRecord]:
 
             if username == "root" and self.target.fs.exists("/root"):
                 # If we got here, 'root' is present both in /var/nstmp and in /root. In such cases, we yield
-                # the 'root' user as having '/root' as a home, not in /var/nstmp.
-                user_home = "/root"
+                # the 'root' user as having '/root' as a home, not in /var/nstmp, as there is no 'nscli_history'
+                # for the root user in /var/nstmp.
+                user_home = self.target.fs.path("/root")
 
-            seen.add(username)
+            seen.add((username, user_home.as_posix() if user_home else None, None))
```

Comment:
Why not just `user_home` as tuple member?

Reply:
`posix_path` instances are unhashable, hence the usage of the string representation